IPs/RSKIP415.md

```diff
@@ -0,0 +1,52 @@
+---
+rskip: 415
+title: Fix pegnatories address derivation from public keys
```

Comment:
Is "fix" the best word to use for this? if I'm not wrong, I recall you told me something regarding using words like "fix," "bug," and "issue." Since skips are shared with the community and impact how users see RSK, shouldn't we use a different term?